lang/po/Firmware_nl.po

```diff
@@ -0,0 +1,1787 @@
+#: messages.c:104
+msgid "Yes"
+msgstr "Ja"
+
+# MSG_WIZARD_QUIT c=20 r=8
+#: messages.c:103
+msgid "You can always resume the Wizard from Calibration -> Wizard."
+msgstr "U kunt de wizard altijd hervatten via Kalibratie -> Wizard."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_SKEW_EXTREME c=20 r=8
+#: ultralcd.cpp:3919
+msgid "XYZ calibration all right. Skew will be corrected automatically."
+msgstr "XYZ-kalibratie in orde. Scheefheid zal automatisch worden gecorrigeerd."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_SKEW_MILD c=20 r=8
+#: ultralcd.cpp:3916
+msgid "XYZ calibration all right. X/Y axes are slightly skewed. Good job!"
+msgstr "XYZ-kalibratie in orde. X / Y-assen zijn licht scheef. Goed gedaan!"
+
+# 
+#: ultralcd.cpp:5197
+msgid "X-correct:"
+msgstr "X-correctie:"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_PERFECT c=20 r=8
+#: ultralcd.cpp:3913
+msgid "XYZ calibration ok. X/Y axes are perpendicular. Congratulations!"
+msgstr "XYZ-kalibratie ok. X / Y-assen staan loodrecht. Gefeliciteerd!"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_BOTH_FAR c=20 r=8
+#: ultralcd.cpp:3897
+msgid "XYZ calibration compromised. Front calibration points not reachable."
+msgstr "XYZ-kalibratie niet gelukt. Voorste kalibratiepunten niet bereikbaar."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_RIGHT_FAR c=20 r=8
+#: ultralcd.cpp:3900
+msgid "XYZ calibration compromised. Right front calibration point not reachable."
+msgstr "XYZ-kalibratie niet gelukt. Rechter voor kalibratiepunt niet bereikbaar."
+
+# MSG_LOAD_ALL c=17
+#: ultralcd.cpp:6236
+msgid "Load all"
+msgstr "Laad alle"
+
+# 
+#: ultralcd.cpp:3879
+msgid "XYZ calibration failed. Bed calibration point was not found."
+msgstr "XYZ-kalibratie mislukt. Bed ijkpunt niet gevonden."
+
+# 
+#: ultralcd.cpp:3885
+msgid "XYZ calibration failed. Front calibration points not reachable."
+msgstr "XYZ-kalibratie mislukt. Voorste kalibratiepunten niet bereikbaar."
+
+# 
+#: ultralcd.cpp:3888
+msgid "XYZ calibration failed. Right front calibration point not reachable."
+msgstr "XYZ-kalibratie mislukt. Rechter voor kalibratiepunt niet bereikbaar."
+
+# 
+#: ultralcd.cpp:3009
+msgid "Y distance from min"
+msgstr "Y afstand van min"
+
+# 
+#: ultralcd.cpp:5198
+msgid "Y-correct:"
+msgstr "Y-correctie:"
+
+# MSG_OFF
+#: menu.cpp:426
+msgid " [off]"
+msgstr " [uit]"
+
+# 
+#: messages.c:57
+msgid "Back"
+msgstr "terug"
+
+# 
+#: ultralcd.cpp:5706
+msgid "Checks"
+msgstr ""
+
+# 
+#: ultralcd.cpp:8080
+msgid "False triggering"
+msgstr "Valse triggering"
+
+# 
+#: ultralcd.cpp:4027
+msgid "FINDA:"
+msgstr ""
+
+# 
+#: ultralcd.cpp:5612
+msgid "Firmware   [none]"
+msgstr "Firmware   [geen]"
+
+# 
+#: ultralcd.cpp:5618
+msgid "Firmware [strict]"
+msgstr "Firmware [strikt]"
+
+# 
+#: ultralcd.cpp:5615
+msgid "Firmware   [warn]"
+msgstr ""
+
+# 
+#: messages.c:87
+msgid "HW Setup"
+msgstr "HW Configuratie"
+
+# 
+#: ultralcd.cpp:4031
+msgid "IR:"
+msgstr ""
+
+# 
+#: ultralcd.cpp:7152
+msgid "Magnets comp.[N/A]"
+msgstr "Magnet. comp.[N/V]"
```

Comment:
could use "Magneet" directly or "Magneten" for plural

Reply:
Limited space